client/client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import requests
+import urlparse
+from django.conf import settings as django_settings
+
+
+class Client(object):
+
+    def __init__(self):
+        self._configure()
+
+    def _configure(self):
+        self.endpoint = "http://172.16.16.24:4568"
```

Comment:
Can we get this endpoint from settings like we are getting the token?

Reply:
Yes it is mentioned in settings but for now we can't access `localhost` from inside `docker` so that's why we are hard coded it here.